### ISSUE TITLE
Skyttel 2024-03-15 is missing default value for missing column

### ISSUE THREAD

**Issue by tft-kaare** [TromsFylkestrafikk/ragnarok-skyttel#8]
During import of 2024-03-15 chunk:
```Import message
SQLSTATE[HY000]: General error: 1364 Field 'OCRConfidenceFront' doesn't have a default value (Connection: mysql, SQL: insert into `skyttel_transaction_trips` (`OperatorReference`, `TourID`, `Departure`, `Registered`, `StopPlaceIDEntry`, `StopPlaceIDExit`, `SignalCode`, `TariffClass`, `LPNFront`, `NationLPNFront`, `LPNRear`, `NationLPNRear`, `SeqLC`, `SeqVideo`, `transaction_id`) values (****)
```

This and probably other fields needs to be more loose on missing data, i.e. `->nullable()`.

**Comment by tft-kaare** [comment]
Also, take a look on the fetch stage of `2024-03-17`. It says: Unable to list contents for 'ReceiptFiles', shallow listing

**Comment by tft-kyrre** [comment]
Import problem (2024-03-15) solved by adding a few default values to the skyttel_transaction_trips table the migration file.

Fetch problem (2024-03-17) was caused by Skyttel returning a empty file list for this chunk date. Seems like ChunkArchive cannot handle this situation (i.e. won't create an empty archive file), so I had to add a dummy file as a workaround.
BTW: Returning **null** from the fetch() function in any sink will result in an exception from SinkHandler.php.
